spark/src/test/scala/org/apache/comet/exec/CometExecSuite.scala

```diff
@@ -519,6 +519,8 @@ class CometExecSuite extends CometTestBase {
           assert(metrics("peak_mem_used").value > 1L)
           assert(metrics.contains("join_time"))
           assert(metrics("join_time").value > 1L)
+          assert(metrics.contains("spill_count"))
+          assert(metrics("spill_count").value == 0)
```

Comment:
Spilling is not triggered. But it should propagate related metrics.

Reply:
Can we also have a test that forces spilling?

Reply:
I will keep trying but wanted to merge these metrics first.